### PR TITLE
Add `is_deleted` column for funding and comment soft deletes

### PR DESCRIPTION
**Pull request by mddilley**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/9209

## Testing
**URL to test:** 
_This PR needs to be tested either locally or on moped-test._

**Steps to test:**

**Comments**
1. Add a comment to a project through a status update in the Summary tab
2. Go to the project's Comments tab and add another comment through the WYSIWYG input
3. In the same tab, delete both comments you added in the Comment list view
4. You should see them disappear from the UI and remain soft deleted when you go to other tabs and return

**Funding**
1. Add a funding to a project on the Funding tab
2. In the same tab, delete the funding source that you just added
4. You should see it disappear from the UI and remain soft deleted when you go to other tabs and return

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] ~Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable~

### CODE REVIEW

File: moped-database/migrations/1653078890676_alter_table_public_moped_proj_notes_add_column_is_deleted/up.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ALTER TABLE "public"."moped_proj_notes" ADD COLUMN "is_deleted" boolean NOT NULL DEFAULT false;
+
+COMMENT ON COLUMN moped_proj_notes.is_deleted IS 'Indicates soft deletion';
+COMMENT ON COLUMN moped_proj_notes.status_id IS 'Deprecated soft deletion column';
```
Comment (chiaberry) [inline moped-database/migrations/1653078890676_alter_table_public_moped_proj_notes_add_column_is_deleted/up.sql:4]: i really like these comments and I am going to start using them ✨

**Review by chiaberry — APPROVED** [review]

File: moped-database/migrations/1654055457679_update_project_list_view_fund_notes_is_del/up.sql [hunk 2]
```diff
@@ -0,0 +1,97 @@
+CREATE OR REPLACE VIEW "public"."project_list_view" AS
```
Comment (frankhereford) [inline moped-database/migrations/1654055457679_update_project_list_view_fund_notes_is_del/up.sql:1]: [Similar to the PR #656](https://github.com/cityofaustin/atd-moped/pull/656#discussion_r889060941), I would suggest a `drop view` statement before the `create`.

File: moped-editor/src/views/projects/projectView/ProjectFundingTable.js [hunk 3]
```diff
@@ -662,6 +662,7 @@ const ProjectFundingTable = () => {
                   ...newData,
                   project_id: projectId,
                   added_by: getDatabaseId(user),
+                  // If no new funding status is selected, the default should be used
```
Comment (frankhereford) [inline moped-editor/src/views/projects/projectView/ProjectFundingTable.js:665]: inline comments that explain intent -- awesome. 💯

**Review by frankhereford — APPROVED** [review]
delete

Looks great @mddilley, thank you!

**Comment by amenity** [comment]
FWIW, after deploying, the "Dashboard" page was taking a while to load so I switched to the "Projects" page and got this error:
Screen Shot 2022-06-08 at 6 57 20 PM

Refreshed and everything loaded fine.

**Review by amenity — APPROVED** [review]
👍🏻 🗑️ 🙏🏻 🚢

File: moped-database/migrations/1654055457679_update_project_list_view_fund_notes_is_del/up.sql (hunk 2 above)
Comment (mddilley) [reply moped-database/migrations/1654055457679_update_project_list_view_fund_notes_is_del/up.sql]: This is now covered in the latest migration pushed here. 🙏

**Comment by mddilley** [comment]
Last commit contains changes from these steps:
1. Pulled in `main`
2. Resolved any conflicts
3. Incorporated changes to the `project_list_view` from my branch into version controlled SQL file
4. Updated view migration in PR to use that version of the `project_list_view`
5. Updated migrations introduced in this branch with fresh timestamps in sequence by milliseconds (to keep local working)

Happy to answer any and all questions about this.

**Review by frankhereford — APPROVED** [review]

**Review by chiaberry — APPROVED** [review]

**Comment by mddilley** [comment]
Just a final summary here, my last commit updated the Dashboard queries for the **My Projects** and **Following** so they use `is_deleted` instead of `status_id` to show only comments that are not soft deleted.

**Review by mateoclarke — COMMENTED** [review]
lizzie-mcguire-delete

**Pull request merged**